src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1373,22 +1378,17 @@ end
 """
     unique(df::AbstractDataFrame; view::Bool=false)
     unique(df::AbstractDataFrame, cols; view::Bool=false)
-    unique!(df::AbstractDataFrame)
-    unique!(df::AbstractDataFrame, cols)
 
 Return a data frame containing only the first occurrence of unique rows in `df`.
 When `cols` is specified, the returned `DataFrame` contains complete rows,
 retaining in each case the first occurrence of a given combination of values
 in selected columns or their transformations. `cols` can be any column
 selector or transformation accepted by [`select`](@ref).
 
-
-For `unique`, if `view=false` a freshly allocated `DataFrame` is returned,
+If `view=false` a freshly allocated `DataFrame` is returned,
 and if `view=true` then a `SubDataFrame` view into `df` is returned.
 
-`unique!` updates `df` in-place and does not support the `view` keyword argument.
-
-See also [`nonunique`](@ref).
+See also: [`unique!`](@ref) [`nonunique`](@ref).
 
 # Arguments
 - `df` : the AbstractDataFrame
```

Comment:
While you're at it, on the line below backticks are missing. Same below.

Reply:
fixed